src/components/feature/feature.njk

```diff
@@ -6,15 +6,15 @@
   {% endif %}
   <div class="rvt-feature__body">
     {% if tag %}
-      <p class="rvt-feature__tag">{{ tag }}</p>
+      <p class="rvt-feature__tag">{% if taglink %}<a href="{{ taglink }}">{% endif %}{{ tag }}{% if taglink %}</a>{% endif %}</p>
     {% endif %}
-    <h2 class="rvt-feature__title">{{ title }}</h2>
+    <h2 class="rvt-feature__title">{% if titlelink %}<a href="{{ titlelink }}">{% endif %}{{ title }}{% if titlelink %}</a>{% endif %}</h2>
     {% if byline %}
       <p class="rvt-feature__byline">{{ byline }}</p>
     {% endif %}
     <p class="rvt-feature__teaser">{{ teaser }}</p>
     {% if meta %}
-      <p class="rvt-feature__meta">{{ meta }}</p>
+      <p class="rvt-feature__meta">{% if metalink %}<a href="{{ metalink }}">{% endif %}{{ meta }}{% if metalink %}</a>{% endif %}</p>
```

Comment:
I don't think this affects the CSS you've written at all, but I would imagine that any of the "meta" items here should be individual links that don't include the pipe as part of the link. So the markup should be able to be rendered as:

```html
<p class="rvt-feature__meta">
  <a href="#2">Meta text</a>  | <a href="#3">Sept. 16, 2019</a>
</p>
```

without the pipe being wrapped in a link.

It might mean making the nunjucks file accept an array of "link" objects and then looping through them to output separate link side the `rvt-feature__meta` element.